addon/hint/show-hint.js

```diff
@@ -42,8 +42,9 @@
     this.widget = null;
     this.debounce = 0;
     this.tick = 0;
-    this.startPos = this.cm.getCursor();
+    this.startPos = this.cm.getCursor("start");
     this.startLen = this.cm.getLine(this.startPos.line).length;
+    if (options && options.somethingSelected == false ) this.startLen = this.startLen - this.cm.getSelection().length;
```

Comment:
By default options.somethingSelected is the same thing than true.

In my case (when I set options.somethingSelected to false), I must remove this.cm.getSelection().length to startLen (like today)